src/main/java/cz/cvut/kbss/study/rest/util/RecordFilterMapper.java

```diff
@@ -45,16 +45,18 @@ public class RecordFilterMapper {
     public static RecordFilterParams constructRecordFilter(String param, String value) {
         return constructRecordFilter(new LinkedMultiValueMap<>(Map.of(param, List.of(value))));
     }
-
+    public static RecordFilterParams constructRecordFilter(MultiValueMap<String, String> params) {
+        final RecordFilterParams result = new RecordFilterParams();
+        return constructRecordFilter(result, new LinkedMultiValueMap<>(params));
+    }
     /**
      * Maps the specified parameters to a new {@link RecordFilterParams} instance.
      *
      * @param params Request parameters to map
      * @return New {@code RecordFilterParams} instance
      */
-    public static RecordFilterParams constructRecordFilter(MultiValueMap<String, String> params) {
+    public static RecordFilterParams constructRecordFilter(final RecordFilterParams result, MultiValueMap<String, String> params) {
```

Comment:
Not sure why we need this.

Reply:
This is here as we did not want to include into query MIN_MODIFIED_DATE and MAX_MODIFIED_DATE .. this should be modified to remove.

Reply:
DO NOT FIX NOW.